react/client/src/components/PageContainer.tsx

```diff
@@ -37,15 +37,15 @@ const PageContainer = ({ match }: PageProps) => {
   // todo: move text into a json for localization
   useEffect(() => {
     switch (currentStep) {
-      case 2:
+      case 'STEP_ENUMS.INTRODUCTION':
```

Comment:
This isn't technically wrong but that's just because the resulting enum is the same as the key :P
Try using the actual enums by importing the map ie `import { STEP_ENUMS } from 'contexts/RoutingProps';`

`Form.tsx` has an example!

Reply:
Thanks! I'll fix it.

Reply:
just pushed a fix. hopefully that does it